Mathlib/Algebra/Order/GroupWithZero/Unbundled.lean

```diff
@@ -1428,12 +1496,22 @@ lemma one_div_nonpos : 1 / a ≤ 0 ↔ a ≤ 0 := one_div a ▸ inv_nonpos
 lemma div_nonpos_of_nonneg_of_nonpos [PosMulMono G₀] (ha : 0 ≤ a) (hb : b ≤ 0) : a / b ≤ 0 := by
   rw [div_eq_mul_inv]; exact mul_nonpos_of_nonneg_of_nonpos ha (inv_nonpos.2 hb)
 
-lemma inv_lt_one_iff₀ [PosMulStrictMono G₀] : a⁻¹ < 1 ↔ a ≤ 0 ∨ 1 < a := by
+variable[PosMulStrictMono G₀] {m n : ℤ}
```

Comment:
```suggestion
variable [PosMulStrictMono G₀] {m n : ℤ}
```
(m and n seem to be used only once; not sure if it's worth putting them in a variable)